data/json/items/magazines.json

```diff
@@ -71,5 +71,218 @@
         "alternatives" : [
             [ "9mm", [ "glockmag" ] ]
         ]
+    },{
+        "type" : "MAGAZINE",
+        "id" : "m1918mag",
+        "//" : "Compatable magazines made for reproductions exist, and the standard-capacity mags are fairly cheap compared to variants.",
+        "price" : 3600,
+        "name" : "BAR 20-round magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A standard capacity magazine for use with the Browning Automatic Rifle, holds 20 rounds.",
+        "material" : "steel",
+        "volume" : 2,
+        "weight" : 198,
+        "to_hit" : -2,
+        "ammo_type" : "3006",
+        "capacity" : 20,
+        "reliability" : 9,
+        "reload_time" : 30
```

Comment:
specify `alternatives` if for `223` if you want this to work with ammo conversion. Probably `stanag30` for both this and `m1918mag_30`